examples/http_routing/main.go

```diff
@@ -41,20 +41,38 @@ type pluginContext struct {
 	// Embed the default plugin context here,
 	// so that we don't need to reimplement all the methods.
 	types.DefaultPluginContext
+
+	diceOverride uint32 // For unit test
```

Comment:
why this is needed?

Reply:
The test was previously overriding the `dice` function directly, but this can't work when the code is within wasm

Reply:
make sense!